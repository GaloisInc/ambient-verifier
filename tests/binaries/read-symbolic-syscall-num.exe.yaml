statechart:
  name: UnsafeOperationViaRead
  root state:
    name: Root
    initial: NormalExecution
    states:
      - name: NormalExecution
        description: |
          Execution before the Weird Machine starts
        transitions:
          - target: WMEntered
            event: EnterWeirdMachine(0x401000)
      - name: WMEntered
        description: |
          The Weird Machine has been entered by clobbering the trusted variable
        type: semi-final(NormalExecution)
