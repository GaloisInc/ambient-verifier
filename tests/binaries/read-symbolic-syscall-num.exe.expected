successful: 0
failed: 2
wmTargets: [0x401000]
fsRoot: tests/filesystems/read
